src/daemon/daemon_rpc.cpp

```diff
@@ -83,6 +81,16 @@ auto make_server(const std::string& server_address, mp::RpcConnectionType conn_t
     return server;
 }
 
+auto server_socket_type_for(const std::string& server_address)
+{
+    if (server_address.find("unix") != std::string::npos)
```

Comment:
Should we compare to `0` instead?

Reply:
Compare what to `0`?  This is making sure the socket type is a unix socket and so it's seeing if `unix` is in the string.

Reply:
Ensure that it _starts_ with `unix`, not that `unix` is somewhere in there. Although I get that should never happen…

Reply:
Yeah, stupid `std::string` doesn't support something like `startsWith()` but I suppose we can do yet another `QString` conversion and do that.

Reply:
Doesn't this work?

```suggestion
    if (server_address.find("unix") == 0)
```

Reply:
Yeah, I think you're right.  I'll try that.

Reply:
Committed separately, but done.